insights/parsers/postfix_conf.py

```diff
@@ -0,0 +1,81 @@
+"""
+PostfixMaster - file ``/etc/postfix/master.cf``
+===============================================
+
+"""
+from insights.core import ConfigParser
+from insights.core.exceptions import SkipComponent
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.postfix_master)
+class PostfixMaster(ConfigParser):
```

Comment:
I'm wondering why `ConfigParser` is used for this parser, as I didn't check that the `parser_doc` is implemented and the super `parser_conent` is called.